### ISSUE TITLE
OCaml 4.06.0

### ISSUE THREAD

**Issue by ejgallego** [camlp5/camlp5#11]
Camlp5 needs an update in order to be compatible with 4.06.0+beta again.

**Comment by roglo** [comment]
Done. Camlp5 version 7.02 released.

**Comment by ejgallego** [comment]
Great thanks, is there any change that you could update the opam file [and thus close #10 ?

**Comment by ejgallego** [comment]
Just bumping:

```
available: [ocaml-version >= "1.07" & ocaml-version <= "4.04.1"]
```
to
```
available: [ocaml-version >= "1.07" & ocaml-version < "4.7.0"]
```
should do the trick.

**Comment by ejgallego** [comment]
PR #12
